examples/RetrievePage/Ballerina.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+org = "wso2"
```

Comment:
@NipunaRanasinghe is this org name is correct?

Reply:
@SasinduDilshara, I think it's fine